pages/common/git-rev-parse.md

```diff
@@ -0,0 +1,12 @@
+# git rev-parse
+
+> Parses a revision.
+> More information: <https://git-scm.com/docs/git-rev-parse>.
+
+- Get the absolute path to the root directory:
+
+`git rev-parse --show-toplevel`
+
+- Get the current branch name:
+
+`git rev-parse --abbrev-ref {{HEAD}}`
```

Comment:
As mentioned in another comment, if HEAD is assumed by default, maybe we can simply omit it from the command.

On this note, I'd say this page is missing an example for addressing a specific (i.e. non-head) revision. Can you please add one?

Reply:
HEAD isn't the default :crying_cat_face: 

And sure, I'll add another example

Reply:
Done in 3923a26b0f1ffe04e37f7fb476d1add4d92bdf87

Reply:
🙏 Thanks!